client/cypress/integration/event-manager/login_spec.js

```diff
@@ -0,0 +1,56 @@
+describe('Logging In', function() {
+
+	context('Should be on login page', function() {
+		it('login page view', function() {
+      cy.visit('/login')
+			cy.get('h4').should('contain', 'Login')
+    })
+	})
+
+	context('Submit button should be disabled on empty fields', function(){
+		it('disabled submit button when both fields are empty', function() {
+			cy.contains('Submit').should('be.disabled')
+		})
+	})
+
+	context('Sould show an error on wrong credentials', function(){
+		it('show an error on wrong credentials', function() {
+			cy.get('input[name=username]').type('hello')
+			cy.get('input[name=password]').type('123')
+			cy.contains('Submit').click()
+			 cy.get('p.error-message')
+        .should('contain', 'No active account found with the given credentials')
+		})
+	})
+
+	context('Submit button should be enabled on filled fields', function(){
+		it('enable submit button when both fields are filled', function() {
+			cy.get('input[name=username]').type('admin')
+			cy.get('input[name=password]').type('1299459ML')
+			cy.contains('Submit').should('be.enabled')
+		})
+	})
+
+	// context('Should redirect to home page on successful login', function(){
```

Comment:
@taleymirza why is this commented out?

Reply:
@mashhoodr because at a time we can only test successful login test case or forget-password/signup routing test case as I mentioned on friday.